src/common/com/intellij/plugins/haxe/ide/folding/HaxeFoldingBuilder.java

```diff
@@ -445,6 +471,23 @@ public String getPlaceholderText(@NotNull ASTNode node) {
 
   @Override
   public boolean isCollapsedByDefault(@NotNull ASTNode node) {
+    final PsiElement psiElement = node.getPsi();
+
+    final CodeFoldingSettings settings = CodeFoldingSettings.getInstance();
+    //final HaxeFoldingSettings haxeSettings = HaxeFoldingSettings.getInstance();
```

Comment:
You don't want to pass through your new stub code?

Reply:
Right now it's just a useless statement, we aren't using this settings right now. But we could add settings for region-block names in other PR as you suggested, .